ipfs/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+description = "Parity IPFS crate"
+name = "parity-ipfs"
+version = "1.6.0"
+license = "GPL-3.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+
+[dependencies]
+ethcore = { path = "../ethcore" }
+ethcore-util = { path = "../util" }
+rlp = { path = "../util/rlp" }
+hyper = { default-features = false, git = "https://github.com/ethcore/hyper" }
+cid = "~0.2.0"
```

Comment:
i believe we use like `cid = "0.2"` everywhere which is almost equivalent to `cid = "~0.2.0"` ?